src/rpc/rawtransaction.cpp

```diff
@@ -94,6 +94,7 @@ UniValue getrawtransaction(const JSONRPCRequest& request)
             "  \"hash\" : \"id\",        (string) The transaction hash (differs from txid for witness transactions)\n"
             "  \"size\" : n,             (numeric) The serialized transaction size\n"
             "  \"vsize\" : n,            (numeric) The virtual transaction size (differs from size for witness transactions)\n"
+            "  \"weight\" : n,           (numeric) The transaction's weight (between vsize*4-3 and vsize*4)\n"
```

Comment:
Would be clearer with a bit of spacing around the `-` operator (as you've done for the help text in decoderawtransaction)

Would also be clearer to return weight first and then define vsize based on weight (since vsize is derived from weight and not vice versa)